src/main/java/org/folio/service/file/reader/LocalStorageCsvSourceReader.java

```diff
@@ -63,7 +62,14 @@ public void init(FileDefinition fileDefinition, ErrorLogService errorLogService,
 
   @Override
   public boolean hasNext() {
-    return iterator.hasNext();
+    boolean result;
```

Comment:
Maybe it is better to set up result as false by default and log error message for exception.

Reply:
Yes, agree.